server/sonar-server/src/main/java/org/sonar/server/usergroups/ws/AbstractGroupUpdate.java

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
I had a shot at delegation instead of inheritance, but indeed I had issues on the constants. I'll have another quick shot at it before merge.